### DISCUSSION TITLE
modelStatus 24

### DISCUSSION THREAD

**Discussion by herbert131** [goecharger/go-eCharger-API-v2#119] [Q&A]
Hi, beim manuellem Phasenwechsel-  bei inaktivem Laden - geht die Wallbox auf  Status 24 ( Ladepause) . Diesen Zustand verlässt sie leider ohne Reboot bei mir nicht mehr. ( SW Vers. 5.5)

**Comment by 0xFEEDC0DE64** [comment]
die aktuellen modelStatus werte ausm source code in der firmware sind:

NotChargingBecauseNoChargeCtrlData=0
NotChargingBecauseOvertemperature=1
NotChargingBecauseAccessControlWait=2
ChargingBecauseForceStateOn=3
NotChargingBecauseForceStateOff=4
NotChargingBecauseScheduler=5
NotChargingBecauseEnergyLimit=6
ChargingBecauseAwattarPriceLow=7
ChargingBecauseAutomaticStopTestLadung=8
ChargingBecauseAutomaticStopNotEnoughTime=9
ChargingBecauseAutomaticStop=10
ChargingBecauseAutomaticStopNoClock=11
ChargingBecausePvSurplus=12
ChargingBecauseFallbackGoEDefault=13
ChargingBecauseFallbackGoEScheduler=14
ChargingBecauseFallbackDefault=15
NotChargingBecauseFallbackGoEAwattar=16
NotChargingBecauseFallbackAwattar=17
NotChargingBecauseFallbackAutomaticStop=18
ChargingBecauseCarCompatibilityKeepAlive=19
ChargingBecauseChargePauseNotAllowed=20
NotChargingBecauseSimulateUnplugging=22
NotChargingBecausePhaseSwitch=23
NotChargingBecauseMinPauseDuration=24
NotChargingBecauseChargeDelay=25
NotChargingBecauseError=26
NotChargingBecauseLoadManagementDoesntWant=27
NotChargingBecauseOcppDoesntWant=28

ich werde es bald in der doku updaten.

warum ist bei dir eine minPauseDuration gesetzt? auto kompatibilität?

**Reply by herbert131** [reply]
keinen Ahnung das ist ja mein Problem ;-)
Ich habe "Standard-Ladeverhalten" gesetzt und sonst nichts umgestellt.
In der Docu konnte ich folgende keys der pause zuordnen ... evtl gibts noch mehr aber ich hab nichts verändert
mcpd = 0
mcpea = null
mptwt = 600000
mpwst = 120000

**Reply by herbert131** [reply]
wenn ich die Phase umschalte geht modelStatus auf 23 und nach wenigen Sekunden auf 24 und verbleibt dort bis zu einen reboot

**Reply by 0xFEEDC0DE64** [reply]
Bitte über den Support klären, wahrscheinlich landet's dann eh bei mir, wir brauchen vorallem die Seriennummer zum Untersuchen übers Backend.

**Comment by iStehle** [comment]
Das ist mein aktuelles Problem mit Evcc, seit einiger Zeit ist das nicht mehr in der Lage Ladungen zu starten, gibt es etwas was ich umstellen kann um nicht mehr dauernd den Go-e Charger neu zu starten?

**Reply by 0xFEEDC0DE64** [reply]
Bitte über den Support klären, wahrscheinlich landet's dann eh bei mir, wir brauchen vorallem die Seriennummer zum Untersuchen übers Backend.
